lib/mixlib/install/generator/powershell/scripts/get_project_metadata.ps1.erb

```diff
@@ -62,16 +65,22 @@ function Get-ProjectMetadata {
   }
   Write-Verbose "Platform Version: $platform_version"
 
-  # TODO: When we ship a 64 bit ruby for Windows.
-  $machine = 'x86_64'
-  Write-Verbose "Machine: $machine"
+  if ($architecture -eq 'auto') {
+    if (!($env:ProgramW6432 -eq $null) -And ($channel -eq 'current')) {
```

Comment:
Rather than relying on the environmental variable, I'd use something like 

```powershell
if ((get-wmiobject win32_operatingsystem).osarchitecture -like '64-bit') -and ($channel -like 'current')){
```